app/main.py

```diff
@@ -14,4 +14,8 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    counts = 0
+    for i in phrase.lower():
```

Comment:
`i` bad variable name

Reply:
Mr. Vasyl, i wrote you solution, please, check it.